test-cases/longevity/longevity-10gb-3h.yaml

```diff
@@ -1,5 +1,5 @@
-test_duration: 180
-stress_cmd: ["cassandra-stress write cl=QUORUM duration=180m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..10000000 -log interval=5"
+test_duration: 10
```

Comment:
why did you change this file ?

Reply:
it was changed for testing purposes. Revert it back. Adding new config parameters